scripts/prepare-machine.ps1

```diff
@@ -381,8 +361,7 @@ if ($IsWindows) {
             Write-Host "[$(Get-Date)] Setting core dump pattern..."
             sudo sh -c "echo -n '%e.%p.%t.core' > /proc/sys/kernel/core_pattern"
             #sudo cat /proc/sys/kernel/core_pattern
-
-            Install-ClogTool "Microsoft.Logging.CLOG2Text.Lttng"
+            Install-ClogTool
```

Comment:
Where's this coming from now that you've deleted it above?

Reply:
I'm debating the following 

a) change my position on checking in code;  due to tool fragmentation in Linux I'm actually starting to like checking in code :)

b) if you want to build the checked in files (clog) you install dotnet, enlist in clog, and build it  - the argument here is that dotnet will likely do a better job with installers than I will - might as well support them

Reply:
That all sounds reasonable. Checked in files for those who just want to build the code. Init the submodule and build clog for those who need to use it to update the checked in files.